keps/sig-multicluster/1645-multi-cluster-services-api/README.md

```diff
@@ -664,6 +672,12 @@ endpoints:
 The `ServiceImport.Spec.IP` (VIP) can be used to access this service from within
 this cluster.
 
+
+#### Multi-network scenario
+One or more clusters in a ClusterSet can be running on a discrete network (a non-flat network). An MCS controller can use the `network.k8s.io` `ClusterProperty` to determine if a cluster in a `ClusterSet` is running on a discrete network. Note that the endpoints of the `EndpointSlice` for a cluster on discrete network may only be representative of the pods backing the multi-cluster service and not the real pod addresses.
```

Comment:
Thinking more about the definition here.  Maybe examples would help?

Consider a cluster with "native" pod IPs.  In that case, the pod network is the same as the node network.  Easy

Consider a cluster with "native" pod IPs, but the pod IPs are in a different subnet.  That's probably also the same as the node network (no visible routing?).

Consider 2 clusters with "native" pod IPs, on different networks where those networks are explicitly peered.  That's more or less the same as the previous case, but "feels" different.

Consider a cluster where the Pods exist on an overlay network (e.g. Flannel).  It's pretty clear that nodes and pods are NOT on the same network.  Which network goes into this property?

Consider a cluster where the Pods are "on" the node network (no encap) but use non-advertised IPs.  Which network goes into this property?

Reply:
@caseydavenport since I know Calico / Canal / Flannel dances in this space.

Reply:
@thockin Thanks for writing down the scenarios, the `network` property refers to the `pod network`. We had some discussion last week and feedback was to change the property name from `network` to something else as `network` is too generic and overloaded. I will update the PR after the discussion, please let me know if you want to be in that meeting.